functions.php

```diff
@@ -989,7 +989,13 @@ function get_jft($sms = false) {
     if ($sms == true) {
         $without_htmlentities = html_entity_decode($trim_results);
         $without_extranewlines = preg_replace("/[$preg_search_lang]+/", "$preg_replace_lang", $without_htmlentities);
-        return $without_extranewlines;
+        $message = sms_chunk_split($without_extranewlines);
+        $finalMessage  = array();
+        for ($i = 0; $i < count($message); $i++) {
+            $jft_message = "(" .($i + 1). " of " .count($message). ")\n" .$message[$i];
```

Comment:
should we only append this if there is more than 1 chunk?

Reply:
probably that would make sense